lib/DiscoveryStreamFeed.jsm

```diff
@@ -1100,204 +1101,92 @@ defaultLayoutResp = {
-          },
+              {
+                "index": 20
+              }
+            ]
+          }
         },
-      ],
-    },
-    {
-      "width": 12,
-      "components": [
         {
-          "type": "CardGrid",
-          "header": {
-            "title": "Business 💼",
-          },
-          "feed": {
-            "embed_reference": null,
-            "url": "https://getpocket.cdn.mozilla.net/v3/firefox/global-recs?topic_id=1&end_time_offset=172800&version=3&consumer_key=$apiKey&locale_lang=en-US&feed_variant=OptimalCuratedLinksForLocaleFeed&model_id=external_time_live",
-          },
+          "type": "Navigation",
           "properties": {
-            "items": 4,
-            "has_numbers": false,
-            "has_images": true,
-          },
-          "styles": {
-            ".ds-header": "margin-top: 4px;",
+            "alignment": "left-align",
+            "links": [
+              {
+                "name": "Must Reads",
+                "url": "https://getpocket.com/explore/must-reads?src=fx_new_tab"
+              },
+              {
+                "name": "Productivity",
+                "url": "https://getpocket.com/explore/productivity?src=fx_new_tab"
+              },
+              {
+                "name": "Health",
+                "url": "https://getpocket.com/explore/health?src=fx_new_tab"
+              },
+              {
+                "name": "Finance",
+                "url": "https://getpocket.com/explore/finance?src=fx_new_tab"
+              },
+              {
+                "name": "Technology",
+                "url": "https://getpocket.com/explore/technology?src=fx_new_tab"
+              },
+              {
+                "name": "More Recommendations ›",
+                "url": "https://getpocket.com/explore/trending?src=fx_new_tab"
+              }
+            ]
           },
-        },
-      ],
-    },
-    {
-      "width": 12,
-      "components": [
-        {
-          "type": "CardGrid",
           "header": {
-            "title": "Science 🔬",
-          },
-          "feed": {
-            "embed_reference": null,
-            "url": "https://getpocket.cdn.mozilla.net/v3/firefox/global-recs?topic_id=7&end_time_offset=172800&version=3&consumer_key=$apiKey&locale_lang=en-US&feed_variant=OptimalCuratedLinksForLocaleFeed&model_id=external_time_live",
-          },
-          "properties": {
-            "items": 4,
-            "has_numbers": false,
-            "has_images": true,
+            "title": "Popular Topics"
           },
           "styles": {
-            ".ds-header": "margin-top: 4px;",
-          },
-          "spocs": {
-            "probability": 1,
-            "positions": [
-              {
-                "index": 3,
-              },
-            ],
-          },
-        },
-      ],
-    },
+            ".ds-navigation": "margin-top: -10px;"
+          }
+        }
+      ]
+    }
   ],
+  "feeds": {},
+  "error": 0,
+  "status": 1
```

Comment:
Are these fields intentionally added now? I believe @ScottDowne explicitly removed them before

Reply:
This is just a copy of the layout JSON from the endpoint, so if it works without them I can remove them!

Reply:
Works without them...removing.